src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void main(String[] args) throws duke.exception.DukeException {
+        final String LOGO = "\n" +
+                " .----------------.  .----------------. \n" +
+                "| .--------------. || .--------------. |\n" +
+                "| |    ______    | || |        __    | |\n" +
+                "| |  .' ____ '.  | || |    _  / /    | |\n" +
+                "| |  | (____) |  | || |   (_)/ /     | |\n" +
+                "| |  '_.____. |  | || |     / / _    | |\n" +
+                "| |  | \\____| |  | || |    / / (_)   | |\n" +
+                "| |   \\______,'  | || |   /_/        | |\n" +
+                "| |              | || |              | |\n" +
+                "| '--------------' || '--------------' |\n" +
+                " '----------------'  '----------------' ";
+        final String DECO_LINE = "____________________________________________________________";
+        final String HELLO_MESSAGE = " Hello! I'm 9%.";
+        final String ASK_MESSAGE = " What can I do for you? (todo/deadline/event)";
+        final String BYE_MESSAGE = " Bye. Hope to see you again soon!";
+        final String ERROR_MESSAGE = " Sorry, I can't recognize your input.";
+        final String ADD_TASK = " Got it. I've added this task:";
+        final String FILE = "./ip/src/main/java/db/tasks.txt";
+
+        Storage storage = new Storage(FILE);
+        ArrayList<Task> inputs = new ArrayList<>(storage.loadTasks());
+
+        System.out.println(LOGO);
+        System.out.println(DECO_LINE);
+        System.out.println(HELLO_MESSAGE);
+        System.out.println(ASK_MESSAGE);
+        System.out.println(DECO_LINE);
+
+        Scanner readinput = new Scanner(System.in);
+        String input = readinput.nextLine();
+
+        while (!input.equals("bye")) {
+            if (input.equals("list")) {
+                System.out.println(DECO_LINE);
+                for (int i = 0; i < inputs.size(); i++) {
+                    System.out.println(" " + (i + 1) + ". " + inputs.get(i).toString());
+                }
+                System.out.println(DECO_LINE);
+            } else if (input.split(" ")[0].equals("done")) {
+                System.out.println(DECO_LINE);
+                try {
+                    int ind = Integer.parseInt(input.split(" ")[1]) - 1;
```

Comment:
Does the variable "ind" indicate the index of the task? Maybe consider using "index" instead.